optaplanner-website-root/content/learn/drl-to-constraint-streams-migration.adoc

```diff
@@ -0,0 +1,447 @@
+# Migrating OptaPlanner score DRL to Constraint Streams
+:jbake-description: Migrate deprecated Score DRL to Constraint Streams
+:toc:
+
+Ever since its introduction back in 2020,
+the https://www.optaplanner.org/docs/optaplanner/latest/constraint-streams/constraint-streams.html[Constraint Streams framework] ("CS") has seen great adoption and has been used to implement any problem you can think of.
+Due to this success, we have finally decided to deprecate its predecessor, the score DRL framework.
+
+## What does deprecating score DRL mean?
+
+Users of OptaPlanner 8 can continue using score DRL without any concern as we will continue maintaining score DRL support for the lifetime of OptaPlanner 8.
+However, OptaPlanner 9 will completely remove all support for score DRL.
```

Comment:
(hard) Mentioning OptaPlanner 9 is dangerous. It gives the perception as we're going to do that soon (which is not true) and creates uncertainty and doubt. I would simply remove this line.